apps/www/lib/redirects.js

```diff
@@ -1298,6 +1298,18 @@ module.exports = [
     source: '/docs/guides/auth/auth-helpers/auth-ui-overview',
     destination: '/docs/guides/auth/auth-helpers/auth-ui',
   },
+  {
+    permanent: true,
```

Comment:
minor nit: on URLs we might resurrect in the short term I like to make the redirects non-permanent, so we can revive them without worrying about cache-busting redirects

Reply:
Good call, I wasn't totally sure on how this worked